src/test/java/org/mastodon/mamut/clustering/ClusterApporientedDemo.java

```diff
@@ -8,10 +8,12 @@
 import com.apporiented.algorithm.clustering.SingleLinkageStrategy;
 import net.miginfocom.swing.MigLayout;
 import org.mastodon.mamut.clustering.ui.DendrogramView;
+import org.mastodon.mamut.clustering.util.Classification;
 
 import javax.swing.JFrame;
 import javax.swing.JPanel;
 import javax.swing.WindowConstants;
+import java.util.Collections;
 import java.util.Random;
 
 public class ClusterApporientedDemo
```

Comment:
A javadoc comment that describes what the class does would be awesome

Reply:
Done here: https://github.com/mastodon-sc/mastodon-deep-lineage/commit/4ee01dbb286adfed644d23b152dc9b523d53b7d4